source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -455,11 +455,6 @@ Filter::StreamOpenState Filter::openStream() {
 
     stream_ = config_->threadLocalStreamManager().store(std::move(stream_object), config_->stats(),
                                                         config_->deferredCloseTimeout());
-    // For custom access logging purposes. Applicable only for Envoy gRPC as Google gRPC does not
-    // have a proper implementation of streamInfo.
-    if (grpc_service_.has_envoy_grpc() && logging_info_ != nullptr) {
```

Comment:
I see you moved the `setClusterInfo(stream_->streamInfo().upstreamClusterInfo())` to logStreamInfoBase, along with setUpstreamHost

It makes sense from ext_proc and good coding practice perspective, but there is the historical reason from logging consumer side that why we want to log the cluster info as earlier as possible.

I will start a thread with internal logging consumer to check this requirement is still true.

Reply:
Thanks for the comments!

I thought the logging will only be retried when the call out finished. So, moving the cluster info setup at the end of the call out should be sufficient.  If that's not the case, we should document it somewhere in the code.

Reply:
@tyxia It looks like the concern of logging cluster info earlier is cleared.  PTAL the PR again.

Reply:
sgtm. Thanks for following up!